Dependencies fetching often fails with volta installation error

Volta Github action installs volta. The first npm command will try to install the right npm and node.js.

Often, this installation process fails with error:

```
Run npm ci
  npm ci
  shell: /usr/bin/bash -e {0}
  env:
    VOLTA_HOME: /opt/hostedtoolcache/volta/1.1.1/x64
Volta error: Could not unpack Node v18.13.0

Please ensure the correct version is specified.
Volta error: Error cause: failed to unpack `/opt/hostedtoolcache/volta/1.1.1/x64/tmp/.tmppc1QRG/node-v18.13.0-linux-x64/bin/node`

Error cause: failed to unpack `/opt/hostedtoolcache/volta/1.1.1/x64/tmp/.tmppc1QRG/node-v18.13.0-linux-x64/bin/node`

Error cause: failed to unpack `node-v18.13.0-linux-x64/bin/node` into `/opt/hostedtoolcache/volta/1.1.1/x64/tmp/.tmppc1QRG/node-v18.13.0-linux-x64/bin/node`

Error cause: Resource temporarily unavailable (os error 11)
Error: Process completed with exit code 126.
```

--- a/.github/workflows/preview.yml
+++ b/.github/workflows/preview.yml
@@ -31,7 +31,9 @@ jobs:
         with:
           fetch-depth: 0
       - name: '[Prepare] Setup Node.js'
-        uses: volta-cli/action@v4
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: 'package.json'
       - name: '[Prepare] Install dependencies'
         run: npm ci
       - name: '[Prepare] Check if a preview already exists'
@@ -83,7 +85,9 @@ jobs:
         with:
           fetch-depth: 0
       - name: '[Prepare] Setup Node.js'
-        uses: volta-cli/action@v4
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: 'package.json'
       - name: '[Prepare] Install dependencies'
         run: npm ci
 

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -38,7 +38,9 @@ jobs:
           fetch-depth: 0
           ref: ${{ github.ref_name }}
       - name: '[Prepare] Setup Node.js'
-        uses: volta-cli/action@v4
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: 'package.json'
       - name: '[Prepare] Extract NPM tag'
         id: npm_tag
         run: p=${{ needs.pre-publish.outputs.prerelease }} && echo "tag=${p:-latest}" >> $GITHUB_OUTPUT
@@ -62,7 +64,9 @@ jobs:
           fetch-depth: 0
           ref: ${{ github.ref_name }}
       - name: '[Prepare] Setup Node.js'
-        uses: volta-cli/action@v4
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: 'package.json'
       - name: '[Prepare] Install dependencies'
         run: npm ci
       - name: '[Build]'

--- a/.github/workflows/test-and-build.yml
+++ b/.github/workflows/test-and-build.yml
@@ -17,7 +17,9 @@ jobs:
         with:
           fetch-depth: 0
       - name: '[Prepare] Setup Node.js'
-        uses: volta-cli/action@v4
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: 'package.json'
       - name: '[Prepare] Install dependencies'
         run: npm ci
 

